src/viewer/modmesh/viewer/PythonInterpreter.cpp

```diff
@@ -133,7 +133,12 @@ PYBIND11_EMBEDDED_MODULE(_modmesh_view, mod)
             [](std::shared_ptr<StaticMesh2d> const & mesh)
             {
                 RApplication * app = RApplication::instance();
-                new RStaticMesh<2>(mesh, app->main()->viewer()->scene());
+                RScene * scene = app->main()->viewer()->scene();
+                for (RStaticMesh<2> * child : scene->findChildren<RStaticMesh<2> *>())
```

Comment:
I already read about `qDeleteAll()` for children removal.  I keep the loop because I do not know which way is better.